apps/ckit-app/src/components/Assets/AssetList/AssetBalance.tsx

```diff
@@ -0,0 +1,35 @@
+import { LoadingOutlined } from '@ant-design/icons';
+import { Typography } from 'antd';
+import React from 'react';
+import { useQuery } from 'react-query';
+import { CkitProviderContainer, WalletContainer } from 'containers';
+import { AssetMeta } from 'hooks';
+import { AssetAmount } from 'utils';
+
+type AssetBalanceProps = Pick<AssetMeta, 'decimal'> & { script: AssetMeta['script'] };
+
+export const AssetBalance: React.FC<AssetBalanceProps> = (props) => {
+  const { script, decimal } = props;
+  const ckitProvider = CkitProviderContainer.useContainer();
+  const { signerAddress } = WalletContainer.useContainer();
+  const query = useQuery(
+    ['queryBalance', script, signerAddress],
```

Comment:
recommend using the object instead of array, since react-query will do shallow equals, object is more clearly

Reply:
fixed